.gitignore

```diff
@@ -56,3 +56,5 @@ examples.txt
 
 # Ignore logs from lerna
 lerna-debug.log
+
+/config/master.key
```

Comment:
I'm pretty certain we don't need this. Let's stick to storing config in the environment, and not use the Rails encrypted store? 🤔